### DISCUSSION TITLE
Issue #2434 status

### DISCUSSION THREAD

**Discussion by steevsachs** [TanStack/query#3091] [Q&A] [answered]
We just ran into #2434 today and I was thrilled to see it's already actively being worked on : ) I see mention of code changes, but don't see a linked PR.  Is this nearing release?  

Any known workarounds in the meantime?

Thanks for all the awesome work!

**Comment by TkDodo** [comment, marked as answer]
why don't you ask on the issue itself? I'm basically just waiting for @JaeYeopHan to contribute the proposed fix :)

**Reply by steevsachs** [reply]
Thank you for the quick response! I didn’t ask on the issue because I didn’t want to interrupt the ongoing discussion : )

I’ll pipe up there with the workarounds question.
